Source/WebCore/crypto/mac/CryptoKeyECMac.cpp

```diff
@@ -252,15 +252,18 @@ RefPtr<CryptoKeyEC> CryptoKeyEC::platformImportSpki(CryptoAlgorithmIdentifier id
     index += oidSize + 1; // Read named curve OID, BIT STRING
     if (keyData.size() < index + 1)
         return nullptr;
-    index += bytesUsedToEncodedLength(keyData[index]) + 1; // Read length, InitialOctet
-
-    if (!doesUncompressedPointMatchNamedCurve(curve, keyData.size() - index))
-        return nullptr;
-
+    index += bytesUsedToEncodedLength(keyData[index])+1; // Read length
     CCECCryptorRef ccPublicKey = nullptr;
-    if (CCECCryptorImportKey(kCCImportKeyBinary, keyData.data() + index, keyData.size() - index, ccECKeyPublic, &ccPublicKey))
-        return nullptr;
-
+    if (!doesUncompressedPointMatchNamedCurve(curve, keyData.size() - index)) {
```

Comment:
I would tend to write if as `if (doesUncompressedPointMatchNamedCurve())` instead of if (! doesUncompressedPointMatchNamedCurve())`